NodeJS / Javascript course: Reconsider or double check following along with the firebase video

### Describe your suggestion

# Suggestion 
Reconsider or double check te assignment of watching and **following** along with the fireship video. 
It would be nice if someone else that has more experience with webpack could try following along with the video  to confirm my findings. 

assignment 1: Watch and follow along with [Fireship’s Module Bundlers Explained video](https://www.youtube.com/watch?v=5IG4UmULyoA) to get a great overview and some hands on experience with webpack.

## The problem 

I've been trying to follow along with the video above. 
I ran into several issues that I don't think are entirely my user error. 

Setup: Mac(M2) and recent installs according to curriculum
Issues I ran into:
## 1. sass-loader issue
- video starts this section [here](https://youtu.be/5IG4UmULyoA?t=369)
- I ran the command `npm install --save-dev css-loader style-loader sass-loader` as shown in the video
- received below error after `npm run build`
> ERROR in ./src/style.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./src/style.scss)
Module Error (from ./node_modules/sass-loader/dist/cjs.js):
Cannot find module 'sass'
- I managed to get rid of the error by separately installing the sass-loader with `npm install sass-loader sass webpack --save-dev   `
- It also throws another less intimidating error after the above fix which is because the creator of the video has used `$color` instead of `$text` in the `style.scss` file
- >ERROR in ./src/style.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./src/style.scss)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Undefined variable.
after fixing the scss file it will compile without errors.

## 2. webpack dev server outdated information (?)
- video section starts [here](https://youtu.be/5IG4UmULyoA?t=478)
- The video will tell you to add this to your webpack config file:
```
devServer: {
    contentBase: path.join(__dirname, "public"),
    port: 9000
  },
```
- after googling, I managed to find that `static` has to be used since web-dev 4 I believe 
- Correct code to add:
```
devServer: {
    static: {
      directory: path.join(__dirname, 'public'),
    },
    compress: true,
    port: 9000,
  },
```

I'm aware I'm a beginner with webpack and some of this might be my own user error, but the searching online made me quite confident about the above.

I believe the errors might provide a negative learning experience for users quite new to Webpack. I have seen someone in discord talking about quitting the video for receiving errors they couldn't grasp.

## Alternative suggestions for this video:
- add notes on the video discrepancies
- remove the part about 'following along` with the video
- add the video to additional resources instead


### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-webpack

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

Manon

### (Optional) Additional Comments

_No response_

--- a/javascript/organizing_your_javascript_code/webpack.md
+++ b/javascript/organizing_your_javascript_code/webpack.md
@@ -15,11 +15,10 @@ This section contains a general overview of topics that you will learn in this l
 
 <div class="lesson-content__panel" markdown="1">
 
-1. Watch and follow along with [Fireship's Module Bundlers Explained video](https://www.youtube.com/watch?v=5IG4UmULyoA) to get a great overview and some hands on experience with webpack.
-2. Go through [this tutorial](https://webpack.js.org/guides/asset-management/) to see examples of using webpack to manage your website's assets.
-3. Read through [this tutorial](https://webpack.js.org/guides/output-management/) to learn how to let webpack automatically manage your index.html and insert your bundle into the page for you!
-4. Finally, the first part of [this tutorial](https://webpack.js.org/guides/development/) talks about source maps, a handy way to track down which source file (index.js, a.js, b.js) an error is coming from when you use webpack to bundle them together. This is essential to debugging bundled code in your browser's DevTools. If the error comes from b.js the error will reference that file instead of the bundle. It also walks through an example of the `--watch` feature you _definitely_ should have taken note of above.
-5. You don't need to do the rest of the webpack tutorials at this time, but take a look at what's offered on the sidebar of their [guides page](https://webpack.js.org/guides/). There are several sweet features that you might want to use in future projects such as code-splitting, lazy-loading, and tree-shaking. Now that you have a handle on webpack's configuration system adding these features is as easy as using the right plugins and loaders!
+1. Go through [this tutorial](https://webpack.js.org/guides/asset-management/) to see examples of using webpack to manage your website's assets.
+2. Read through [this tutorial](https://webpack.js.org/guides/output-management/) to learn how to let webpack automatically manage your index.html and insert your bundle into the page for you!
+3. Finally, the first part of [this tutorial](https://webpack.js.org/guides/development/) talks about source maps, a handy way to track down which source file (index.js, a.js, b.js) an error is coming from when you use webpack to bundle them together. This is essential to debugging bundled code in your browser's DevTools. If the error comes from b.js the error will reference that file instead of the bundle. It also walks through an example of the `--watch` feature you _definitely_ should have taken note of above.
+4. You don't need to do the rest of the webpack tutorials at this time, but take a look at what's offered on the sidebar of their [guides page](https://webpack.js.org/guides/). There are several sweet features that you might want to use in future projects such as code-splitting, lazy-loading, and tree-shaking. Now that you have a handle on webpack's configuration system adding these features is as easy as using the right plugins and loaders!
 
 </div>
 

